src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
Le 01/03/2016 8:32, Daniel Bünzli a écrit :
>> @@ -823,10 +826,13 @@ module ConfigSyntax = struct
>>                 (kind,criterion)::List.remove_assoc kind 
>> t.solver_criteria }
>>    let with_solver t solver = { t with solver = Some solver }
>> 
>> +  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
> 
>> I would rather use this value when initialising the original config 
>> file
> 
> I would rather not. Why can't this be on `opam.ocaml.org` which is
> community managed ?

I have no strong opinion on where this backup should be hosted, this 
could probably be hosted on `opam.ocaml.org`, but what's wrong in 
OCamlPro offering this service and some sysadmin time ? Anyway, the 
default URL might be changed by the end-user.